adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -153,6 +155,20 @@ public PaymentRequest createAuthorizationRequest(final String merchantAccount,
         return paymentRequest;
     }
 
+    public PaymentsDetailsRequest create3DPaymentsRequest(final String paymentData, final String md, final String paRes) {
+
+        PaymentsDetailsRequest paymentsDetailsRequest = new PaymentsDetailsRequest();
+
+        HashMap<String, String> details = new HashMap<>();
```

Comment:
here we could move this key-value pair to library (like we do in "set3DRequestData"), since it could be handy for other library users